eZ/Publish/Core/REST/Server/Input/Parser/Criterion/DateMetadata.php

```diff
@@ -8,14 +8,27 @@
  */
 namespace eZ\Publish\Core\REST\Server\Input\Parser\Criterion;
 
+use eZ\Publish\Core\REST\Common\Exceptions\Parser;
 use eZ\Publish\Core\REST\Common\Input\BaseParser;
 use eZ\Publish\Core\REST\Common\Input\ParsingDispatcher;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\DateMetadata as DateMetadataCriterion;
 
 /**
  * Parser for ViewInput Criterion.
  */
 class DateMetadata extends BaseParser
 {
+    /** @var array */
+    public $availableOperators = [
```

Comment:
I was thinking about changing this value or inheritance, but after rethink it can be changed I guess